packages/native/src/styled.js

```diff
@@ -0,0 +1,9 @@
+import { StyleSheet } from 'react-native'
+import { createStyled } from '@emotion/primitives-core'
+
+/**
+ * a function that returns a styled component which render styles in React Native
+ */
+let styled = createStyled(StyleSheet)
```

Comment:
this is also a good candidate for PURE

Reply:
are there any side-effects of using PURE here ?